README.md

```diff
@@ -37,17 +37,20 @@ Currently, chDB only supports `query` function, which is used to execute SQL and
 
 ```python
 import chdb
-res = chdb.query('select version()', 'CSV'); print(str(res.get_memview().tobytes()))
+res = chdb.query('select version()', 'Pretty'); print(res.data())
```

Comment:
Not intent to let user use `data()` function. As it will do an extra data copy of the result. 
But as it is much shorter than `get_memview().tobytes()`, I think this is ok.
Most Python users care more about less code than little performance difference.